engine/src/stack.h

```diff
@@ -947,6 +950,10 @@ class MCStack : public MCObject, public MCMixinObjectHandle<MCStack>
     bool isscriptonly(void) const { return m_is_script_only; }
     void setasscriptonly(MCStringRef p_script);
     
+    // BWM-2017-08-16: [[ Bug 17810 ]] Get/set line endings for imported script-only-stack.
+    MCStringLineEndingStyle getlineencodingstyle(void) const { return m_line_encoding_style; }
+    void setlineencodingstyle(MCStringLineEndingStyle p_line_encoding_style);
```

Comment:
It is probably worth putting the set definition inline here as well - as it doesn't do anything significant beyond update an internal var.